proxy/src/cancellation.rs

```diff
@@ -29,9 +29,7 @@ use crate::tls::postgres_rustls::MakeRustlsConnect;
 
 type IpSubnetKey = IpNet;
 
-const CANCEL_KEY_TTL: i64 = 1_209_600; // 2 weeks cancellation key expire time
-const REDIS_SEND_TIMEOUT: std::time::Duration = std::time::Duration::from_millis(10);
-const BATCH_SIZE: usize = 8;
+const CANCEL_KEY_TTL: i64 = 604_800; // 1 week cancellation key expire time
```

Comment:
Isn't it 2 weeks for a reason? Because we usually let proxies run that long?

Reply:
Forgot that it will stay for 1 week more after the release. Thanks :)